### PR TITLE
Add OSET PL to approved licenses

### PR DESCRIPTION
**Pull request by paultag**
To be merged if the OPL is approved

### CODE REVIEW

**Comment by paultag** [comment]
cc @richardfontana

File: licenses/manual/opl.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[
+    {
+        "id": "OPL",
+        "links": [
+            {
+                "note": "OSET Foundation Page",
+                "url": "http://www.osetfoundation.org/public-license#"
```
Comment (tianon) [inline licenses/manual/opl.json]: Trailing `#` intentional?  (mostly harmless cosmetic defect :see_no_evil:)

Comment (paultag) [inline licenses/manual/opl.json]: alas, no. thanks @tianon :D

Comment (paultag) [inline licenses/manual/opl.json]: force push'd an update

File: licenses/manual/opl.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+[
+    {
+        "id": "OPL",
```
Comment (paultag) [inline licenses/manual/opl.json]: Add v 2.1

**Pull request merged**